x/pricefeed/client/cli/tx.go

```diff
@@ -52,17 +55,23 @@ func GetCmdPostPrice(cdc *codec.Codec) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			expiryInt, ok := sdk.NewIntFromString(args[2])
```

Comment:
note to author: no need to use `NewIntFromString` when you don't need an `sdk.Int` -- better to use the stdlib.